extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -284,7 +284,16 @@ public function getWikiVariables() {
 			);
 		}
 
-		$wikiVariables['navigation'] = $navigation;
+		if ( empty( $wgEnableGlobalNav2016 ) ) {
+			$wikiVariables['navigation'] = $navigation;
+		} else {
+			$navigationNodes = ( new GlobalNavigationHelper() )->getMenuNodes2016();
+			$wikiVariables['hubsLinks'] = $navigationNodes['hubs'];
+			$wikiVariables['explore'] = $navigationNodes['explore'];
+			$wikiVariables['localNav'] = $navigation;
+			$wikiVariables['enableGlobalNav2016'] = $wgEnableGlobalNav2016;
```

Comment:
Can we put all this data in `$wikiVariables['navigation']`? It is navigation data after all. Also would be nice to have all this logic in the `getNavigation` method.